AcademicReward/Database/GroupDatabase.cs

```diff
@@ -0,0 +1,134 @@
+﻿using AcademicReward.Resources;
+using Npgsql;
+using System.Collections.ObjectModel;
+
+namespace AcademicReward.Database
+{
```

Comment:
Can we update the comments in this class? Some are referring to gathering tasks even though this is for group stuff?